client/zeth/core/ownership.py

```diff
@@ -24,7 +24,9 @@
 
 class OwnershipKeyPair:
     """
-    Key-pair for ownership proof
+    Key-pair for ownership proof. This represents the public and private
+    components of a ZethAddress used in note commitments in the joinsplit
```

Comment:
Sticking to the language of the specs, the ownership keypair is a tuple made of:
1. the "paying key" (apk) from the "Payment address"
2. the "spending key" (ask) from the "Private address"

Reply:
adjusted comment

Reply:
See: https://raw.githubusercontent.com/clearmatics/zeth-specifications/master/zeth-protocol-specification.pdf p25 and p26. The payment address is public and this is a tuple of `apk`, `pkenc`. I fixed the documentation in #301

Reply:
Sure.  (I always get confused by this use of "address" to cover both private and public components).